packages/dds/tree/src/test/simple-tree/schemaFactory.spec.ts

```diff
@@ -109,8 +114,38 @@ describe("schemaFactory", () => {
 	it("instanceof", () => {
 		const schema = new SchemaFactory("com.example");
 
-		const config = new TreeConfiguration(schema.number, () => 5);
+		class A extends schema.object("A", {}) {}
+		class B extends schema.object("B", {}) {}
+		const C = schema.object("C", {});
+		const StructuralArray = schema.array(A);
+		const NominalArray = schema.array("D", A);
+
+		const a = new A({});
+		assert(a instanceof A);
+		assert(a instanceof TreeNode);
+		assert(!(a instanceof B));
 
+		const c = new C({});
+		assert(c instanceof C);
```

Comment:
this can was allowed before this change, but produced the opposite result (false), which would cause incorrect narrowing of TypeScript types at runtime.